src/index.js

```diff
@@ -180,21 +154,17 @@ $.ajax({
     }
 
     // Set the current responses counter
-    var responseCount = 0
-    for (i = 0; i < responses.length; i++) {
+    let responseCount = 0
+    for (let i = 0; i < responses.length; i++) {
       question = questions[i]
       switch (question.input.type) {
         case 'checkbox':
         case 'radio':
         case 'inputs':
-          if (!!responses[i] && !!responses[i].join('')) {
-            responseCount++
-          }
+          responses[i] && responseCount++
```

Comment:
The tests may pass but it's not the same.
responses could be an empty array. An empty array is not valid response.